executor/insert_common.go

```diff
@@ -948,7 +952,12 @@ func (e *InsertValues) collectRuntimeStatsEnabled() bool {
 func (e *InsertValues) batchCheckAndInsert(ctx context.Context, rows [][]types.Datum, addRecord func(ctx context.Context, row []types.Datum) error) error {
```

Comment:
Also need to add some runtime stats in `batchUpdateDupRows` methods and `InsertExec.exec` method?